src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/DeleteInstanceV6.sql

```diff
@@ -267,6 +263,12 @@ BEGIN
         AND CF.StudyInstanceUid = DI.StudyInstanceUid
         AND CF.SeriesInstanceUid = DI.SeriesInstanceUid
         AND CF.SopInstanceUid = DI.SopInstanceUid
+    
+    -- Delete FileProperties of instance
+    DELETE FP
+    FROM dbo.FileProperty as FP
+    INNER JOIN @deletedInstances AS DI
+    ON FP.Watermark = DI.Watermark
```

Comment:
Do we already have `instanceKey` such that we can join on it and avoid a new index?

Reply:
Same thing with the join above